### PR TITLE
Migrate cloudwatch_metric_alarm* modules and tests

### PR DESCRIPTION
**Pull request by alinabuzachis**
Depends-On: https://github.com/ansible/zuul-config/pull/443

Migrate cloudwatch_metric_alarm* modules and tests

### CODE REVIEW

**Review by gravesm — APPROVED** [review]

**Review by jillr — APPROVED** [review]

File: tests/integration/targets/cloudwatch_metric_alarm/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+---
```
Comment (abikouo) [inline tests/integration/targets/cloudwatch_metric_alarm/vars/main.yml:1]: This file can be deleted

**Review by abikouo — APPROVED** [review]
LGTM!

File: tests/integration/targets/cloudwatch_metric_alarm/vars/main.yml (hunk 1 above)
Comment (alinabuzachis) [reply tests/integration/targets/cloudwatch_metric_alarm/vars/main.yml]: Done, thanks!

**Review by goneri — APPROVED** [review]

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Pull request merged**